src/test/create-user.test.ts

```diff
@@ -0,0 +1,62 @@
+import { setup } from '../setup';
+import * as request from 'supertest';
+import { expect } from 'chai';
+import { getRepository } from 'typeorm';
+import { User } from '../entity/User';
+
+before(() => {
+  setup();
+});
+
+async function createUserMutation(variables: any) {
+  const query = `
+    mutation createUser($data: UserInput!) {
+      createUser(data: $data) {
+        id
+        name
+        email
+      }
+    }`;
+  return request('localhost:4001')
+    .post('/')
+    .send({ query, variables: { data: variables } });
+}
+
+describe('create-user test', function () {
+  it('password not validated', async function () {
```

Comment:
Geralmente enunciamos o teste com `should + condição`. Exemplo: `should give an error if password is not valid`.

Reply:
bacana saber, vou alterar